OpenSim/Simulation/Model/OffsetFrame.h

```diff
@@ -123,17 +128,18 @@ class  OffsetFrame : public C {
     This transform is stored via the translation and orientation
     properties of this object.
     
-    @param transform  The transform between this frame and its parent frame.
+    @param offset   The transform between this frame and its parent frame.
     */
-    void setOffsetTransform(const SimTK::Transform& offsetTransform);
+    void setOffsetTransform(const SimTK::Transform& offset);
 
 protected:
     /** The transform X_GF for this OffsetFrame, F, in ground, G.*/
-    const SimTK::Transform
+    SimTK::Transform
         calcGroundTransform(const SimTK::State& state) const override;
 
     /** Extend how OffsetFrame determines its base Frame. */
     const Frame& extendFindBaseFrame() const final;
+    /** Extend how OffsetFrame determines its transform in its base Frame. */
     SimTK::Transform extendFindTransformInBaseFrame() const final;
```

Comment:
Sorry, Ajay -- I know you didn't like it but per discussion in #299 and simbody/simbody#311 this and the previous one should be `override final` to be consistent.